README.Rmd

```diff
@@ -29,35 +29,43 @@ knitr::opts_chunk$set(
-environment. Environments built with Nix contain R and all the required packages
-that you need for your project: there are currently more than 80.000 pieces of
-software available through the Nix package manager, including the entirety of
-CRAN and Bioconductor packages. It is also possible to install older releases of
-packages, or install packages from GitHub.
-
-The Nix package manager is extremely powerful: not only it handles all the
-dependencies of any package extremely well, it is also possible with it to
-reproduce environments containing old releases of software. It is thus possible
-to build environments that contain R version 4.0.0 (for example) to run an old
-project originally developed on that version of R.
+`{rix}` is an R package that leverages [Nix](https://nixos.org/),
+a powerful package manager focusing on reproducible builds. With Nix, it is
+possible to create project-specific environments that contain a project-specific
+version of R and R packages (as well as other tools or languages, if needed).
+You can use `{rix}` and Nix to replace `{renv}` and Docker with one single tool.
+Nix is an incredibly useful piece of software for ensuring reproducibility of
+projects, in research or otherwise. For example, it allows you run web 
+applications like Shiny apps or `{plumber}` APIs in a controlled environment.
+
+Nix has a fairly high entry cost though. Nix is a complex piece of software that
+comes with its own programming language, which is also called Nix. Its purpose
+is to solve a complex problem: defining instructions how to build software
+packages and manage configurations in declarative way. This makes sure that 
+software gets installed in fully reproducible manner, on any operating system or
+hardware.
+
+`{rix}` provides functions to help you write and deploy Nix expressions (written
+in the Nix language). These expressions will be the inputs for the Nix package
+manager, to build sets of software packages and provide them in a reproducible
+development environment. These environments can be used for interactive data
+analysis, or reproduced when running pipelines in CI/CD systems. On the 
+[Nixpkgs collection](https://github.com/nixos/nixpkgs), there are currently more
+than 80.000 pieces of software available through the Nix package manager. Using
+{rix}, you can define and build isolated R environments through Nix package
+manager with ease. Like this, environments contain R and all the required 
+packages that you need for your project. You can also add any other software
+tool available. The Nix R ecosystem currently includes the entirety of CRAN and
+Bioconductor packages. Like with any other programming language and software, it
+is also possible to install older releases of R packages, or install packages
+from GitHub at defined states.
+
+The Nix package manager is extremely powerful. Not only does it handle all the
+dependencies their builds extremely well in deterministic manner. It is also 
```

Comment:
Not only does it handle all the dependencies of any package extremely well in a deterministic manner, it is also possible with it to reproduce environments containing old releases of software.

I would write it like this, just one sentences. But here we repeat that it handles old software releases. Maybe we should remove it above?

Reply:
fair point, i'll remove the repetition.

Reply:
back to one sentence sounds good.